src/ocean/task/Task.d

```diff
@@ -638,6 +630,12 @@ unittest
 {
     // test exception forwarding semantics
 
+
+    // if scheduler was initialized before it will have one pending task from
+    // thrown exception - drop it before proceeding:
```

Comment:
This sentence is hard to understand, because the "before" is ambiguous. I think it should be more like:

```d
\\ If the scheduler was initialized before, it will have one pending task from
\\ the thrown exception. Drop it before proceeding.
```

Reply:
Updated